src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -149,6 +150,9 @@ def prepare_or_run_pipeline(
         Raises:
             RuntimeError: If a connector is used that does not return a
                 `boto3.Session` object.
+
```

Comment:
@christianversloot Python 3.8 Ubuntu still fails ... no idea why... maybe this extra line causes the error in the CI?